inst/config/default.yaml

```
@@ -59,6 +59,17 @@ reportingTemplate: onlyRes
 historic: NULL
 
 
+# slurm settings ---------------------------------------------------------------
+
+### slurm quality of service (= QOS)
+slurmQOS: NULL
```

```suggestion
slurmQOS: default
```
Okay, now I'm unsure if my suggestions make sense. You probably though about this already and have good reason to do it the way you suggested. Then please ignore my bothering.

---

See above - I'm also unsure ;-)